### ISSUE TITLE
Copy from functionality not working with ACF fields

### ISSUE THREAD

**Issue by matt-banc** [qtranslate/qtranslate-xt#876]
Hi thanks for continuing the development of this plugin..
I have an issue with the "copy from" not saving ACF fields after copying from other language (fields type is qtranslate).
If I could get some direction on how to fix it...
below is my config details:
{
  "default_language": "gb",
  "enabled_languages": [
    "us",
    "gb"
  ],
  "flag_location": "plugins/qtranslate-xt-master/flags/",
  "language_name": {
    "us": "English US",
    "gb": "English UK"
  },
  "locale": {
    "us": "en_US",
    "gb": "en_GB"
  },
  "locale_html": {
    "us": "en_US",
    "gb": "en_GB"
  },
  "not_available": {
    "us": "Sorry, this entry is only available in %LANG:, : and %.",
    "gb": "Sorry, this entry is only available in %LANG:, : and %."
  },
  "date_format": {
    "us": "%A %B %e%q, %Y",
    "gb": "%A %B %e%q, %Y"
  },
  "time_format": {
    "us": "%I:%M %p",
    "gb": "%I:%M %p"
  },
  "flag": {
    "us": "us.png",
    "gb": "gb.png"
  },
  "url_mode": 2,
  "use_strftime": 3,
  "filter_options_mode": 0,
  "language_name_case": 0,
  "detect_browser_language": false,
  "hide_untranslated": false,
  "show_displayed_language_prefix": false,
  "show_alternative_content": true,
  "hide_default_language": false,
  "use_secure_cookie": false,
  "header_css_on": true,
  "header_css": ".qtranxs_flag_en {background-image: url(https://mydomain/wp-content/plugins/qtranslate-xt-master/flags/gb.png); background-repeat: no-repeat;}\n.qtranxs_flag_us {background-image: url(https://mydomain/wp-content/plugins/qtranslate-xt-master/flags/us.png); background-repeat: no-repeat;}\n",
  "text_field_filters": [],
  "front_config": {
    "all-pages": {
      "filters": {
        "text": {
          "widget_title": "20",
          "widget_text": "20",
          "the_title": "20",
          "category_description": "20",
          "list_cats": "20",
          "wp_dropdown_cats": "20",
          "term_name": "20",
          "get_comment_author": "20",
          "the_author": "20",
          "tml_title": "20",
          "term_description": "20"
        },
        "term": {
          "cat_row": "0",
          "cat_rows": "0",
          "wp_get_object_terms": "0",
          "single_cat_title": "0",
          "single_tag_title": "0",
          "single_term_title": "0",
          "the_category": "0",
          "get_term": "0",
          "get_terms": "0",
          "get_category": "0"
        },
        "url": {
          "author_feed_link": "10",
          "author_link": "10",
          "get_comment_author_url_link": "10",
          "post_comments_feed_link": "10",
          "day_link": "10",
          "month_link": "10",
          "year_link": "10",
          "page_link": "10",
          "post_link": "10",
          "category_link": "10",
          "category_feed_link": "10",
          "tag_link": "10",
          "term_link": "10",
          "the_permalink": "10",
          "feed_link": "10",
          "tag_feed_link": "10"
        }
      }
    }
  },
  "term_name": {
    "Finances": {
      "us": "Financial",
      "gb": "Finances"
    }
  },
  "disable_client_cookies": false,
  "ignore_file_types": [
    "gif",
    "jpg",
    "jpeg",
    "png",
    "svg",
    "pdf",
    "swf",
    "tif",
    "rar",
    "zip",
    "7z",
    "mpg",
    "divx",
    "mpeg",
    "avi",
    "css",
    "js",
    "mp3",
    "mp4",
    "apk"
  ],
  "url_info": {
    "cookie_enabled": true,
    "cookie_lang_front": "gb",
    "cookie_lang_admin": "gb",
    "scheme": "https",
    "host": "wp.intouchnetworks.com",
    "path": "/wp-admin/admin-ajax.php",
    "query": "action=admin_debug_info",
    "path-base": "",
    "path-base-length": 0,
    "wp-path": "/wp-admin/admin-ajax.php",
    "language_neutral_path": true,
    "http_referer": "https://mydomain/wp-admin/options-general.php?page=qtranslate-xt",
    "referer_admin": true,
    "doing_front_end": false,
    "lang_cookie_admin": "gb",
    "lang_admin": "gb",
    "language": "gb",
    "set_cookie": false
  },
  "language": "gb",
  "editor_mode": 0,
  "highlight_mode": 1,
  "auto_update_mo": true,
  "hide_lsb_copy_content": false,
  "lsb_style": "Simple_Tabs.css",
  "config_files": [
    "./i18n-config.json"
  ],
  "admin_config": {
    "post": {
      "pages": {
        "post.php": "",
        "post-new.php": ""
      },
      "anchors": {
        "post-body-content": {
          "where": "first last"
        }
      },
      "forms": {
        "post": {
          "fields": {
            "title": [],
            "excerpt": [],
            "attachment_caption": [],
            "attachment_alt": [],
            "view-post-btn": {
              "encode": "display"
            },
            "wp-editor-area": {
              "jquery": ".wp-editor-area"
            },
            "gallery-caption": {
              "jquery": ".gallery-caption",
              "encode": "none"
            },
            "wp-caption-text": {
              "jquery": ".wp-caption-text",
              "encode": "display"
            },
            "img-alt": {
              "jquery": "img",
              "attrs": [
                "alt"
              ],
              "encode": "display"
            }
          }
        },
        "wpbody-content": {
          "fields": {
            "wrap-h1": {
              "jquery": ".wrap h1",
              "encode": "display"
            },
            "wrap-h2": {
              "jquery": ".wrap h2",
              "encode": "display"
            }
          }
        }
      },
      "js-exec": {
        "post-exec": {
          "src": "./admin/js/post-exec.min.js"
        }
      }
    },
    "options-general": {
      "preg_delimiter": "#",
      "pages": {
        "options-general.php": "^(?!.*page=[^=&]+).*$"
      },
      "forms": {
        "options": {
          "fields": {
            "blogname": [],
            "blogdescription": [],
            "head-title": {
              "jquery": "head title",
              "encode": "display"
            }
          }
        }
      }
    },
    "widgets": {
      "pages": {
        "widgets.php": ""
      },
      "anchors": {
        "widgets-right": {
          "where": "before after"
        }
      },
      "forms": {
        "widgets-right": {
          "fields": {
            "widget-title": {
              "jquery": "input[id^='widget-'][id$='-title']"
            },
            "widget-text-text": {
              "jquery": "textarea[id^='widget-text-'][id$='-text']"
            },
            "in-widget-title": {
              "jquery": "span.in-widget-title",
              "encode": "display"
            }
          }
        }
      },
      "js-exec": {
        "widgets-exec": {
          "src": "./admin/js/widgets-exec.min.js"
        }
      }
    },
    "edit-term": {
      "pages": {
        "term.php": "",
        "edit-tags.php": "action=edit"
      },
      "forms": {
        "edittag": {
          "fields": {
            "name": [],
            "description": [],
            "parent": {
              "encode": "display"
            }
          }
        }
      }
    },
    "edit-tags": {
      "preg_delimiter": "#",
      "pages": {
        "edit-tags.php": "^(?!.*action=edit).*$"
      },
      "anchors": {
        "posts-filter": {
          "where": "before after"
        }
      },
      "forms": {
        "addtag": {
          "fields": {
            "tag-name": [],
            "tag-description": [],
            "parent": {
              "encode": "display"
            }
          }
        },
        "col-left": {
          "fields": {
            "tagcloud": {
              "jquery": ".tagcloud > a",
              "encode": "display"
            }
          }
        },
        "the-list": {
          "fields": {
            "row-title": {
              "jquery": ".row-title",
              "encode": "display"
            },
            "description": {
              "jquery": ".description",
              "encode": "display"
            }
          }
        }
      },
      "js-exec": {
        "edit-tags-exec": {
          "src": "./admin/js/edit-tags-exec.min.js"
        }
      }
    },
    "nav-menus": {
      "pages": {
        "nav-menus.php": "action=edit|menu=\\d+|^$"
      },
      "anchors": {
        "menu-to-edit": {
          "where": "before after"
        }
      },
      "forms": {
        "update-nav-menu": {
          "fields": {
            "title": {
              "jquery": "[id^=edit-menu-item-title-]"
            },
            "attr-title": {
              "jquery": "[id^=edit-menu-item-attr-title-]"
            },
            "description": {
              "jquery": "[id^=edit-menu-item-description-]"
            },
            "span.title": {
              "jquery": "span.menu-item-title",
              "encode": "display"
            },
            "link-to-original": {
              "jquery": ".link-to-original >a",
              "encode": "display"
            }
          }
        },
        "side-sortables": {
          "fields": {
            "label.title": {
              "jquery": "label.menu-item-title",
              "encode": "display"
            },
            "accordion-section-title": {
              "jquery": "h3.accordion-section-title",
              "encode": "display"
            }
          }
        }
      },
      "js-exec": {
        "nav-menus-exec": {
          "src": "./admin/js/nav-menus-exec.min.js"
        }
      }
    },
    "all-pages": {
      "filters": {
        "text": {
          "admin_title": "20"
        }
      }
    }
  },
  "custom_i18n_config": [],
  "custom_fields": [],
  "custom_field_classes": [],
  "post_type_excluded": [],
  "translator": {},
  "qtrans_compatibility": false,
  "i18n-cache": {
    "admin_page_configs": {
      "": {
        "filters": {
          "text": {
            "admin_title": "20"
          }
        }
      }
    }
  }
}

**Comment by eva-bi** [comment]
@herrvigg
i have same above issue, i using wp 5.5.3 & qtranslate-xt 3.9.1
sorry ... when will you fix it?
